staff/david-casas/bash/README.md

```diff
@@ -0,0 +1,92 @@
+# BASH
+
+Command-line interface commands typical in BASH.
+
+## pwd (path to working directory)
+
+```sh
+$ pwd
+```
+
+
+## ls (list/files folder)
+
+```sh
+$ ls
+```
+
+## ls -l(list files with details)
+
+```sh
+$ ls -l
+
+drwxr-xr-x 1 david 197609       0 Oct 12 09:15  Videos/
+-rw-r--r-- 1 david 197609 1175552 Oct 11 05:10  ntuser.dat.LOG1
+-rw-r--r-- 1 david 197609   20480 Oct 11 05:10  ntuser.dat.LOG2
+-rw-r--r-- 1 david 197609      20 Oct 11 05:10  ntuser.ini
+drwxr-xr-x 1 david 197609       0 Nov  7 04:57 'prueba acceso'/
+drwxr-xr-x 1 david 197609      
+
+```
+## ls -a (hidden folders)
+
+```sh
+$ ls -a
+ntuser.dat.LOG1
+ntuser.dat.LOG2
+ntuser.ini
+```
+## touch (creates an empty file)
+
+```sh
+$ touch hello.world
+```
+## mkdir(creates a directory / folder)
+
+```sh
+$ mkdir
+
+```
+## cd (change directory)
+
+```sh
+$ cd workspace
+```
+
+## mv (move / rename file or folder)
+
+```sh
+$ mv
```

Comment:
Here you have to put the complete command.
If it is to rename: $mv file1 file2
If it is to move: $mv file1 /folder1/file1

Reply:
done as well